extensions/wikia/DesignSystem/node_modules/design-system/dist/css/styles.css

```diff
@@ -1 +1 @@
index:1}.wds-global-navigation__dropdown-controls,.wds-global-navigation__modal-controls{align-items:center;height:55px;justify-content:flex-end;width:100%}.wds-global-navigation__dropdown-controls{display:none}@media only screen and (min-width:768px){.wds-global-navigation__dropdown-controls{display:flex}}.wds-global-navigation__modal-controls{display:none}@media only screen and (max-width:767px){.wds-global-navigation{padding:0 3px 0 12px}.wds-global-navigation__modal-controls{display:flex}}.wds-global-navigation__modal-control-user{position:relative}.wds-global-navigation__modal{background-color:#f2f5f5;height:calc(100% - 55px);left:0;overflow-y:scroll;position:fixed;top:55px;width:100%}.wds-global-navigation.wds-is-modal-opened{z-index:900}.wds-global-navigation.wds-is-modal-opened .wds-global-navigation__modal-control{align-items:center;display:flex;justify-content:center;height:100%;padding:10px}.wds-global-navigation.wds-is-modal-opened .wds-global-navigation__modal-control-anon,.wds-global-navigation.wds-is-modal-opened .wds-global-navigation__modal-control-search,.wds-global-navigation.wds-is-modal-opened .wds-global-navigation__modal-control-user{display:none}.wds-global-navigation.wds-is-modal-opened .wds-global-navigation__modal-control-close{display:block}.wds-global-navigation__modal-control{align-items:center;box-sizing:border-box;display:flex;justify-content:center;height:100%;padding:10px}.wds-global-navigation__modal-control .wds-avatar__image,.wds-global-navigation__modal-control svg{transition:border-color .3s,fill .3s}.wds-global-navigation__modal-control-search{position:relative}.wds-global-navigation__modal-control-close svg,.wds-global-navigation__modal-control-search svg{fill:#00d6d6}.wds-global-navigation__modal-control-close{display:none}.wds-global-navigation__modal-control-close svg,.wds-global-navigation__modal-control:hover svg{fill:#fff}.wds-global-navigation__modal-control:hover .wds-avatar__image{border-color:#fff;fill:#fff}.wds-global-navigation__links{display:none;font-size:18px;font-weight:700;height:100%}@media only screen and (min-width:768px){.wds-global-navigation__links{display:flex}}@media only screen and (max-width:1279px){.wds-global-navigation__links{font-size:16px}}.wds-global-navigation__link{text-transform:uppercase;transition:color .3s,font-size .2s,margin-left .2s}.wds-global-navigation__link,.wds-global-navigation__start-a-wiki{align-items:center;color:#00d6d6;display:flex;height:100%;margin-left:30px;text-decoration:none}.wds-global-navigation__link:visited,.wds-global-navigation__start-a-wiki:visited{color:#00d6d6!important}.wds-global-navigation__link:hover,.wds-global-navigation__start-a-wiki:hover{color:#fff!important}.wds-global-navigation__link .wds-dropdown__toggle-chevron,.wds-global-navigation__start-a-wiki .wds-dropdown__toggle-chevron{fill:#ff6a64}.wds-global-navigation__link .wds-global-navigation__link-button,.wds-global-navigation__start-a-wiki .wds-global-navigation__link-button{
```

Comment:
newline

Reply:
This file is installed with npm, I'm not touching it.